code/client/munkilib/updatecheck.py

```diff
@@ -2581,6 +2581,15 @@ def getmanifest(partialurl, suppress_errors=False):
     munkicommon.display_debug2('Manifest base URL is: %s', manifestbaseurl)
     munkicommon.display_detail('Getting manifest %s...', manifestdisplayname)
     manifestpath = os.path.join(manifest_dir, manifestname)
+
+    # Create the folder the manifest shall be stored in
+    destinationdir = os.path.dirname(manifestpath)
+    try:
+        os.makedirs(destinationdir)
+    except OSError:
+        if not os.path.isdir(destinationdir):
+            raise
```

Comment:
Why bother with the except if we are only going to raise? Either use munkicommon.display_error() to give a more helpful error message, use ManifestException to do something similar, or don't bother at all.  The exception will be raised anyway...